### ISSUE TITLE
Modlishka setup in windows issue

### ISSUE THREAD

**Issue by Ayushraj1** [drk1wi/Modlishka#78]
Hello Piotr,
Thanks for revolutionary.
But I can't setup and run this tool properly on local host in windows.
Can you explain the process what I've to do to to run this tool properly.
Because the most time I get encountered with error like loopback.modlishks.io:443 send invalid response.
AND if I manage to run url, then It does not capture users credential and after signing into gmail ,it redirects to original Google.Com URL.

Can you add the process to setup in Windows In WiKI or here.
Please reply as soon as possible .

**Comment by drk1wi** [comment]
Hi, hm, the standard usage scenario is on WIKI, but it also should work regardless of the OS.  The issue with credential capture is probably related to your regex config.

**Comment by Ayushraj1** [comment]
Idk what is regex, regular expression?
I am.not using anything like regex.

**Comment by drk1wi** [comment]
yes, for the control panel to work you will have to create a regular expression that will extract the credentials. otherwise all of the usernames/passwords and tokens are always logged to a log file

**Comment by Ayushraj1** [comment]
Yes, but I checked log file there is nothing captured and saved there.
Can you tell me how to define regex
And there is no dist/proxy in modlishka file, so how can I run custom template.
